packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -0,0 +1,168 @@
+
+We are also currently working on introducing more robust theming capabilities in a future PatternFly major release. Going forward, all future themes will include a light and dark theme.
+
+There are a couple of ways to enable dark theme in your products. You can: 
+
+1. Add `patternfly-theme-dark.css` to a page and add the `.pf-theme-dark` class to the page's `<html>` element, or 
+2. Apply the `.pf-theme-dark` class to the browser's `prefers-color-scheme` media query. 
+
+For information regarding the shipping and implementation processes for dark theme, as well as general advice for its use read our [dark theme handbook.](/developer-resources/dark-theme-handbook)
+
+## React 18 support
+
+We are committed to supporting the 2 most recent versions of React for PatternFly to keep up with the environments that our users work in. To this end, we are pleased to announce that PatternFly 5 is compatible with both React 17 and React 18.
+
+## Versioned CSS variables and class names
+
+We wanted to ensure that our variable naming conventions were clear and consistent going forward with our future release cycles. To keep things organized and clear, we updated our CSS variables and class names so that they are explicitly associated with the version of PatternFly that they align to. For PatternFly 5, we added a prefix of  `v5` to a number of our variable and class names.
+
+For more details, refer to [our upgrade guide](/get-started/upgrade#review-and-update-variable-and-class-names) and our [updated CSS classes and variables](developer-resources/global-css-variables) documentation.
+
+## Event handler parameter consistency updates
+
+Previously, our event handler parameters weren’t consistently ordered. To meet industry standards, and get everything on the same page, we added event parameters to all of our callbacks and reordered handlers to consistently list the event as the first parameter. Additionally, we made sure that all of our event handlers now pass back an event. This change will make it easier to incorporate third party libraries, especially in the case of form building tools, like Formik.
+
+Our [codemods](/get-started/upgrade#run-our-codemods) should take care of most of these changes, but if your callback function is defined at the class level you will need to manually update your code.
+
+## Changes to React icons
+
+To improve the consistency of icon usage across projects adopting PatternFly, we are no longer using inline styles, color, or size for icons. Now, React icons come with a built-in class `pf-v5-svg`, which gives an icon a default height, width, and color.
```

Comment:
the color is still set on the svg itself from react icons via the `fill` attribute - https://github.com/patternfly/patternfly-react/blob/6d3f867f64c75fd7b3fe0f896d40ce712a07bc7c/packages/react-icons/src/createIcon.tsx#L46

We now set the 3 css properties `width`, `height`, and `vertical-align`.

```suggestion
To improve the consistency of icon usage across projects adopting PatternFly, we are no longer using inline styles, color, or size for icons. Now, React icons come with a built-in class `pf-v5-svg`, which gives an icon a default height, width, and vertical alignment.
```

Or you could reword this a bit to be more clear, and say "which gives an icon a default value for `height`, `width`, and `vertical align`.